wd_helper.au3

```diff
@@ -636,6 +636,199 @@ Func _WD_FrameLeave($sSession)
 	Return SetError(__WD_Error($sFuncName, $_WD_ERROR_Success), 0, $sValue)
 EndFunc   ;==>_WD_FrameLeave
 
+; #FUNCTION# ====================================================================================================================
+; Name ..........: _WD_FrameList
+; Description ...: get list of documents (window.top and all frames) from current webdriver context/tab
+; Syntax ........: _WD_FrameList($sSession[, $bReturnAsArray = True[, $sFilter = ''[, $bReturnHTML = False]]])
+; Parameters ....: $sSession            - Session ID from _WD_CreateSession
+;                  $bReturnAsArray      - By default on True function will return array, other wise string
+;                  $sFilter             - RegExp pattern to check if desired content exist in frame document HTML source, if used it returns only frames with diesired content
+;                  $bReturnHTML         - On true function will return frame document HTML source
+; Return values .: Success - array or string
+;                  Failure - "" (empty string) and sets @error to one of the following values:
+;                  - $_WD_ERROR_GeneralError
+;                  - $_WD_ERROR_Exception
+;                  - $_WD_ERROR_InvalidExpression
+;                  - $_WD_ERROR_NotFound
+; Author ........: mLipok
+; Modified ......:
+; Remarks .......: using $sFilter will set $bReturnHTML
+; Related .......: __WD_FrameList_Internal, _WD_FrameEnter, _WD_GetFrameCount
```

Comment:
Related function list shouldn't include internal functions.

Reply:
fixed in:
[further changes](https://github.com/Danp2/au3WebDriver/pull/362/commits/3bdf5437e1f1792e68f9d69804f593ce4cd6ac52)